includes/definitions/discovery/vrp.yaml

```diff
@@ -336,16 +344,14 @@ modules:
                     descr: '{{ $entPhysicalName }} Rx'
                     entPhysicalIndex: '{{ $index }}'
                     entPhysicalIndex_measured: ports
-                    index: 'lane-rx-{{ $index }}'
```

Comment:
Are you sure you need to change the index here ? it will break all the historical data (RRD)

Reply:
I'm open to discussion on this. For me, this was the way _not_ to lose historical data. The power level discovered in `rx-{{ $index }}` was fine and could be continued, just the power levels were broken.

Reply:
Yep, but for devices (like mine) where we use the index you are changing, then all the data is lost ... 
I would say the best would be to
- keep the 2 different indexes 
- change the conditions that decide when one or the other index is used
- publish a breaking-change message to explain which devices will change from one index to the other (as far as you know).